### PR TITLE
Podpeople available as roundstart ft. minor stat changes

### PR DESCRIPTION
**Pull request by Stokeling9701**
Essentially it allows podmen to be roundstart along with editing them so they take slightly more fire damage and more damage from plant-b-gone

 :cl: Daniel0Mclovin
add: Podpeople to roundstart
tweak: Modified podpeople heatmod from 1.25 to 1.5
tweak: plant-b-gone now does roughly 7 damage per tick on podpeople
/:cl:

[Reason]:
It'd be nice to see pod people around a bit more. Opens up more roleplay opportunities and another creature to see on board. Only supports basic color changes.

### CODE REVIEW

**Comment by Wilchenx** [comment]
stop

File: code/modules/mob/living/carbon/human/species_types/podpeople.dm [hunk 1]
```diff
@@ -10,8 +10,11 @@
 	burnmod = 1.25
 	heatmod = 1.5
 	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/plant
-	disliked_food = NONE
-	liked_food = NONE
+	disliked_food = MEAT
+	liked_food = VEGETABLES | FRUIT
+	toxic_food = TOXIC | RAW
```
Comment (Wilchenx) [inline code/modules/mob/living/carbon/human/species_types/podpeople.dm]: Citadel just changed it to none in another PR

Comment (Stokeling9701) [reply code/modules/mob/living/carbon/human/species_types/podpeople.dm]: ah i didnt know that, why was it changed?

Comment (Ratvar) [reply code/modules/mob/living/carbon/human/species_types/podpeople.dm]: "stealthly nerfed lizrds cannot eat donk pockets" blahblah

Comment (silicons) [reply code/modules/mob/living/carbon/human/species_types/podpeople.dm]: set to none

File: code/modules/mob/living/carbon/human/species_types/podpeople.dm [hunk 2]
```diff
@@ -10,8 +10,11 @@
...
+	roundstart = 1
```
Comment (silicons) [inline code/modules/mob/living/carbon/human/species_types/podpeople.dm]: TRUE

File: code/modules/mob/living/carbon/human/species_types/podpeople.dm [hunk 3]
```diff
@@ -38,14 +41,17 @@
...
+		H.take_overall_damage(2,5)
+
 
 /datum/species/pod/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
 	if(chem.id == "plantbgone")
-		H.adjustToxLoss(3)
+		H.adjustToxLoss(5)
 		H.reagents.remove_reagent(chem.id, REAGENTS_METABOLISM)
 		return 1
+		H.confused = max(H.confused, 1)
```
Comment (silicons) [inline code/modules/mob/living/carbon/human/species_types/podpeople.dm]: make this above the return

File: code/modules/mob/living/carbon/human/species_types/podpeople.dm [hunk 4]
```diff
@@ -38,14 +41,17 @@
...
+	if(H.nutrition < NUTRITION_LEVEL_STARVING + 55)
+		H.take_overall_damage(2,5)
+
 
 /datum/species/pod/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
 	if(chem.id == "plantbgone")
-		H.adjustToxLoss(3)
+		H.adjustToxLoss(5)
 		H.reagents.remove_reagent(chem.id, REAGENTS_METABOLISM)
 		return 1
```
Comment (silicons) [inline code/modules/mob/living/carbon/human/species_types/podpeople.dm]: and while your'e at it, return TRUE

**Comment by Kevak** [comment]
I approve.

**Review by deathride58 — COMMENTED** [review]
You should probably nerf the heal_overall_damage and adjustToxLoss in spec_life to around 0.05. Keeping them at TG's value of 1 would mean podpeople wouldn't have to take a single trip to medbay in most cases. Nerfing their passive regeneration to 0.05 would mean antags won't have to break a bunch of lights just to crit a podperson in hand-to-hand combat, and it'd mean podpeople would actually have to visit medbay if they stay in the dark too long.

**Comment by Stokeling9701** [comment]
Whenever podpeople seem to be in the dark for too long they nearly go into instant crit, their self regen wont help it and being dragged into a lighted area doesnt seem to stop the massive amounts of burn and brute damage that kills them

**Comment by Kevak** [comment]
Why are podpeople getting burn/brute from the dark? They should get oxy prolly. The darkness does not burn after all. And light is literally their equivalent of oxygen.

**Comment by LetterJay** [comment]
Still unsure about species with stats.

**Comment by Wilchenx** [comment]
@LetterJay we have slimepeople already

**Comment by Stokeling9701** [comment]
they take brute and burn normally, i didnt change that at all.

File: code/modules/mob/living/carbon/human/species_types/podpeople.dm [hunk 5]
```diff
@@ -34,18 +37,21 @@
...
 			H.nutrition = NUTRITION_LEVEL_FULL
 		if(light_amount > 0.2) //if there's enough light, heal
-			H.heal_overall_damage(1,1)
-			H.adjustToxLoss(-1)
-			H.adjustOxyLoss(-1)
+			H.heal_overall_damage(0.7)
+			H.adjustOxyLoss(-0.5)
+
+	if(H.nutrition < NUTRITION_LEVEL_STARVING + 55)
+		H.adjustOxyLoss(5,5) //can eat to negate this unfortunately
```
Comment (silicons) [inline code/modules/mob/living/carbon/human/species_types/podpeople.dm]: are you sure this is how you use oxygen damage?

File: code/modules/mob/living/carbon/human/species_types/podpeople.dm [hunk 6]
```diff
@@ -34,18 +37,21 @@
...
+		H.adjustToxLoss(3,0)
```
Comment (silicons) [inline code/modules/mob/living/carbon/human/species_types/podpeople.dm]: are you sure this is how you use toxins damage?

File: code/modules/mob/living/carbon/human/species_types/podpeople.dm [hunk 7]
```diff
@@ -34,18 +37,21 @@
 		if(H.nutrition > NUTRITION_LEVEL_FULL)
 			H.nutrition = NUTRITION_LEVEL_FULL
 		if(light_amount > 0.2) //if there's enough light, heal
-			H.heal_overall_damage(1,1)
-			H.adjustToxLoss(-1)
-			H.adjustOxyLoss(-1)
+			H.heal_overall_damage(0.7)
```
Comment (silicons) [inline code/modules/mob/living/carbon/human/species_types/podpeople.dm]: are you sure this is how you use heal overall damage?

**Comment by silicons** [comment]
image
this is how you use those procs

**Review by silicons — APPROVED** [review]
Code: Working, probably good, I'm assuming you tested all this.
Balance except for regen values: Good I guess.
REGEN VALUES: Imo contrary to the haters having to wait 40 seconds to heal a single point of damage? Why is it even there at that point, but whatever. I'd recommend a system that lets them heal faster (0.5-1.5) if they haven't taken damage in <x> amount of time but I don't care on that.

**Comment by Stokeling9701** [comment]
Regen will be altered to regen hopefully aroumd 1 damage every 3ish ticks. Maybe 4

**Comment by LetterJay** [comment]
We'll go with it for now. If it's not liked it can be removed.

**Pull request merged**